.github/workflows/tests.yml

```diff
@@ -27,13 +27,74 @@ jobs:
       - name: Rebase to current master
         run: git rebase origin/master
 
+      - name: Prepare
+        run: |
+          # # override GitHub's bind mount from host, we don't want anything from there and it interferes with ssh
+          # export HOME=$(getent passwd $(id -u) | cut -f6 -d:)
+
+          # Put the /secrets in place which we'll mount to the tasks-container
+          mkdir -p ~/secrets
+          echo '${{ secrets.COCKPITUOUS_TOKEN }}' > ~/.config/github-token
+          echo '${{ secrets.SSH_KNOWN_HOSTS }}' > ~/secrets/known_hosts
+          echo '${{ secrets.SINK_SSH_CONFIG }}' > ~/secrets/ssh-config
+          echo '${{ secrets.SINK_ID_RSA }}' > ~/secrets/id_rsa
+          echo '${{ secrets.SINK_ID_RSA_PUB }}' > ~/secrets/id_rsa.pub
+          sudo chmod 400 ~/secrets/*
+          # change permissions to user in cockpit-tasks
+          # TODO sudo needed?
+          sudo chown 1111:1111 ~/secrets/*
+
+          # install dependencies needed to use bots stuff in the host container
+          sudo pip3 install pika
```

Comment:
"container" == "GitHub workflow host VM"?

I'd rather we let the webhook container handle the AMQP bits, and only inject HTTP requests to the webhook container. Or perhaps, to work with this version, it can do something like `podman exec -i cockpituous-tasks sh -exc 'cd cockpit-project/bots; ./tests-scan ...`?

Reply:
yea that's probably better :)